.github/workflows/scale-test-gce.yaml

```diff
@@ -200,27 +204,43 @@ jobs:
         run: |
           cilium status --wait
       
-      - name: Display CL2 config
-        run: |
-          cat ${{ env.cl2_config }}
-      
       - name: Run CL2
+        id: run-cl2
         working-directory: ./perf-tests/clusterloader2
+        # --enable-exec-service=false to reduce number of pods so 100 pods can fit in node
+        # POD_STARTUP_LATENCY_THRESHOLD=60s so the test doesn't fail, currently we have ~30s pods startup latency
         run: |
           mkdir ./report
+          echo POD_STARTUP_LATENCY_THRESHOLD: 60s >> ./testoverrides.yaml
           go run ./cmd/clusterloader.go \
             -v=4 \
-            --testconfig=${{ env.cl2_config }} \
+            --testconfig=./testing/node-throughput/config.yaml \
+            --testoverrides=./testoverrides.yaml \
+            --enable-exec-service=false \
             --provider=gce \
             --enable-prometheus-server \
             --tear-down-prometheus-server=false \
             --report-dir=./report \
             --kubeconfig=$HOME/.kube/config
 
+      - name: Get sysdump
+        if: ${{ always() && steps.run-cl2.outcome != 'skipped' }}
+        run: |
+          cilium status
+          cilium sysdump --output-filename cilium-sysdump-final
+
       - name: Cleanup cluster
-        if: ${{ always() && steps.deploy.outcome != 'skipped' }}
+        if: ${{ always() && steps.deploy-cluster.outcome != 'skipped' }}
         run: |
           ./kops delete cluster \
             ${{ env.cluster_name }} \
             --state ${{ secrets.GCP_PERF_KOPS_STATE_STORE }} \
             --yes
+
+      - name: Export results and sysdump to GS bucket
+        if: ${{ always() && steps.run-cl2.outcome != 'skipped' }}
+        run: |
+          # BUILD_ID needs to be an integer
+          BUILD_ID="$(date +%s)"
+          EXPORT_DIR="${{ secrets.GCP_PERF_RESULTS_BUCKET }}/logs/scale-test-${{ github.ref_name }}/${BUILD_ID}/artifacts/"
+          gsutil -m cp -R cilium-sysdump-final.zip ./perf-tests/clusterloader2/report/* $EXPORT_DIR
```

Comment:
Can we include the repository details in the export dir path? That way we can separate results that may have been run from a fork, so as not to taint the data:

```
EXPORT_DIR="${{ secrets.GCP_PERF_RESULTS_BUCKET }}/logs/scale-test-${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.ref_name }}/${BUILD_ID}/artifacts/"
```

Can we also add the following in the artifact directory somewhere? I was thinking it would be useful to have some reference to:

* Cluster name
* Commit SHA
* URL to the run on GHA

This extra info would be useful for debugging.

Reply:
I wouldn't add a repository - other repositories shouldn't be allowed to push results here, they can specify their own bucket instead. 
Good call with commit sha and URL, it definitely makes sense to add it. I've also added CL2 output too.

Reply:
Do you mind adding a comment in the workflow explaining that folks need to use their own bucket if running from a fork? Just to make it more explicit?

Reply:
But it's also true for other variables like `GCP_PERF_SA_KEY`, `GCP_PERF_PROJECT_ID`, or `GCP_PERF_KOPS_STATE_STORE` as we don't want to interfere with ci scale tests and this is expected also when running any other CI in a fork I think?